app/controllers/submit.js

```diff
@@ -188,8 +188,10 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
 
     hasFile: Ember.computed.or('file', 'selectedFile'),
 
-    // True if fields have been changed
-    hasDirtyFields: Ember.computed.or('uploadChanged', 'basicsChanged', 'disciplineChanged'),
+    // True if fields have been changed and a file has been selected
+    hasDirtyFields: Ember.computed('hasFile', 'uploadChanged', 'basicsChanged', 'disciplineChanged', function() {
+        return this.get('hasFile') && (this.get('uploadChanged') || this.get('basicsChanged') || this.get('disciplineChanged'));
```

Comment:
return isAdding && hasFile || this.get('uploadChanged') || this.get('basicsChanged') || this.get('disciplineChanged');